src/libs/API.js

```diff
@@ -33,6 +47,20 @@ function write(command, apiCommandParameters = {}, onyxData = {}) {
     SequentialQueue.push(request);
 }
 
+/**
+ * API.makeRequestWithSideEffects() is for blocking commands that have "side-effects" (such as authentication or
```

Comment:
Can we add a link to the doc on how to use this? maybe API.md or something. Or is there nowhere where this is described?

Reply:
I kind of think it's OK to just leave this in the code and I don't really think it's necessary to put in the API.md (just my opinion though). I think the best place to document specific methods is in the JS docs and the readme is better for defining broad best-practices.

Reply:
Ah I took the same description for `makeRequestWithSideEffects` from the design doc [here](https://docs.google.com/document/d/1AF1iTabP3WUSpNPE8-C2NJ7jYSBIISViRRBZSvXNSug/edit#bookmark=id.v6op8xamv7ea), but not sure if will be redundant since it will be the same information.

Also, we don't have information about it in the API.md, but I think it will be a good idea to add more info about it.

Reply:
Oops I didn't notice Tim's comment after I posted my comment.

> I think the best place to document specific methods is in the JS docs and the readme is better for defining broad best-practices.
+1